### DISCUSSION TITLE
How do I do a one to many relationship from order to products

### DISCUSSION THREAD

**Discussion by Dandrizzy** [supabase/supabase#22535] [Questions]
I have a little challenge with my e-commerce project where a customer selects the product they want and checkout from the cart page. After checking out how do I make a one to many relationships such that on my order table I'd have order_id and that order_id would point to my product table of the selected items in that table that consists of all the products available. Upon querying the order_id I should get a list of the the selected products the customer made please can someone help me out

**Comment by GaryAustin1** [comment]
You can have an order/products table where you have order_id, product_id foreign keys.  Then you just populate a row per product in the order.
To use it in the API make sure both keys are part of the primary key.

https://supabase.com/docs/guides/database/joins-and-nesting#many-to-many-joins
